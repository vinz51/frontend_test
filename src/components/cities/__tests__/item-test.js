import City from "api/models/city";
import CityItem from "../item";
import React from "react";
import renderer from "react-test-renderer";

describe("City item", () => {
  describe("Snaphosts", () => {
    function getMyTreeComponent(props = {}) {
      return renderer
        .create(
          <CityItem
            city={new City({ city: "Paris", state: "Ile de France" })}
            {...props}
          />
        )
        .toJSON();
    }
    it("renders", () => {
      expect(getMyTreeComponent()).toMatchSnapshot();
    });

    it("renders with as prop", () => {
      expect(getMyTreeComponent({ as: "div" })).toMatchSnapshot();
    });
  });
});
