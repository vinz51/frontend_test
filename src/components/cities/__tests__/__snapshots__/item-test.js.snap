// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City item Snaphosts renders 1`] = `
<li
  className="sc-AxhUy kBCMWZ"
>
  <h2
    className="sc-AxirZ sc-AxiKw hYWAVI"
  >
    Paris
  </h2>
  <h2
    className="sc-AxirZ sc-AxhCb dspALo"
  >
    Ile de France
  </h2>
</li>
`;

exports[`City item Snaphosts renders with as prop 1`] = `
<div>
  <h2
    className="sc-AxirZ sc-AxiKw hYWAVI"
  >
    Paris
  </h2>
  <h2
    className="sc-AxirZ sc-AxhCb dspALo"
  >
    Ile de France
  </h2>
</div>
`;
